File: encrypt.py
#!/usr/bin/env python

from Cryptodome.Cipher import AES
import binascii

key       = b"AES256Key-32Characters1234567890"
iv        = b"123456789012"
plaintext = b"message to be encrypted"

aesCipher = AES.new(key, AES.MODE_GCM, iv)
ciphertext, authTag = aesCipher.encrypt_and_digest(plaintext)

print("ciphertext = ", binascii.hexlify(ciphertext))
print("tag = ", binascii.hexlify(authTag))

File: encrypt.pl
#!/usr/bin/env perl

use strict;
use feature qw(say);
use Crypt::AuthEnc::GCM qw(gcm_encrypt_authenticate gcm_decrypt_verify);

my $key       = "AES256Key-32Characters1234567890";
my $iv        = "123456789012";
my $plaintext = "message to be encrypted";

my ($ct, $tag) = gcm_encrypt_authenticate('AES', $key, $iv, '', $plaintext);
say "ciphertext = " . unpack('H*', $ct);
say "tag = " . unpack('H*', $tag);
